src/syncPersistCache.js

```diff
@@ -0,0 +1,52 @@
+import { CachePersistor } from 'apollo-cache-persist';
+import Persistor from 'apollo-cache-persist/Persistor';
+import Storage from 'apollo-cache-persist/Storage';
+
+export default function(options) {
```

Comment:
Would it be possible to avoid default export as this can mess up commonjs imports vs ts imports.

Reply:
We need to link that in main index file to be able to use it with published package

Reply:
Yep, sounds good. I was trying to follow the same pattern as with persistCache.ts, but that makes sense.